assets/js/main.js

```diff
@@ -12,6 +12,8 @@ kaboom({
 
 loadRoot("../../public/scene_background_images/");
 loadSprite("instructions-page", "instructions.png");
+loadSprite("game-page", "game.png");
+loadSprite("quit-page", "game_over.png");
```

Comment:
Issue (Blocking): I don't think these are required for you to add here because Kyle is working on them, so I recommend just discarding any changes made to the `main.js` file.